### PR TITLE
Product category delete and get all

### PR DESCRIPTION
**Pull request by Elisheva1234**

### CODE REVIEW

File: src/main/java/com/sapred/ordermanagerred/controller/ProductCategoryController.java [hunk 1]
```diff
@@ -1,20 +1,45 @@
...
 import com.sapred.ordermanagerred.service.ProductCategoryService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 @RestController
 @RequestMapping("/ProductCategory")
```
Comment (moriyahazan) [inline src/main/java/com/sapred/ordermanagerred/controller/ProductCategoryController.java:15]: path should be start with a lower letter

File: src/main/java/com/sapred/ordermanagerred/service/ProductCategoryService.java [hunk 2]
```diff
@@ -1,35 +1,49 @@
...
+                throw new NoPermissionException("You do not have the appropriate permission to delete product category");
+
+            productCategoryRepository.deleteById(id);
+
+    }
+    public List<ProductCategoryDto> getAllCategory(String token) {
+        RoleOptions role = jwtToken.getRoleIdFromToken(token);
+        String companyIdFromToken = jwtToken.getCompanyIdFromToken(token);
+        if (role == RoleOptions.CUSTOMER)
+            throw new NoPermissionException("You do not have the appropriate permission to delete product category");
```
Comment (moriyahazan) [inline src/main/java/com/sapred/ordermanagerred/service/ProductCategoryService.java:44]: Please fix the validation

**Review by moriyahazan — APPROVED** [review]
Good Job
some small comments please fix

**Comment by Elisheva1234** [comment]
I changed, check the changes for me

**Pull request merged**